libs/core-ui/src/lib/util/datasetUtils/getPropertyValues.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { DatasetCohortColumns } from "../../DatasetCohortColumns";
+import { IDataset } from "../../Interfaces/IDataset";
+import { ModelTypes } from "../../Interfaces/IExplanationContext";
+import { IsBinary, IsMulticlass } from "../ExplanationUtils";
+import { MulticlassClassificationEnum } from "../JointDatasetUtils";
+
+// get property values for the selected data point
+export function getPropertyValues(
+  indexes: number[],
+  property: string,
+  dataset: IDataset,
+  modelType?: ModelTypes
+): unknown[] {
+  if (property === DatasetCohortColumns.Index) {
+    return indexes;
+  }
+  if (property === DatasetCohortColumns.PredictedY) {
+    const predictedYs = dataset.predicted_y;
+    if (predictedYs) {
+      return indexes.map((index) => {
+        return predictedYs[index];
+      });
+    }
+  }
+  const classNames = dataset.class_names;
+  if (
+    classNames &&
+    property === DatasetCohortColumns.ProbabilityY + classNames[0]
```

Comment:
for my learning, what is classNames[0] and className[1] here? will classNames always be length of 2?

Reply:
good question! should handle not only the first two classNames. let me update,
classNames example
![image](https://user-images.githubusercontent.com/76190371/225091580-5c5f866c-3329-4f7e-a599-66cce3d0ccbf.png)